google-http-client-gson/pom.xml

```diff
@@ -67,11 +67,6 @@
       <artifactId>google-http-client-test</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
```

Comment:
Note: this is inherited from `google-http-client-test` and we inherit from a base test class.